.env

```diff
@@ -0,0 +1,31 @@
+# Environment for local demo network.
+# This file is meant to work with docker-compose.yaml
+
+RUST_LOG=info,libp2p=off
+RUST_LOG_FORMAT=full
+
+L1_BLOCK_TIME_SEC=3
+
+# Internal port inside container
+ESPRESSO_WEB_SERVER_PORT=40000 
+ESPRESSO_ORCHESTRATOR_PORT=40001
+ESPRESSO_CONSENSUS_SERVER_PORT=40002
+ESPRESSO_DA_SERVER_PORT=40003
+ESPRESSO_SEQUENCER_DA_SERVER_URL=http://da-server:$ESPRESSO_WEB_SERVER_PORT
+ESPRESSO_SEQUENCER_CONSENSUS_SERVER_URL=http://consensus-server:$ESPRESSO_WEB_SERVER_PORT
+ESPRESSO_SEQUENCER_ORCHESTRATOR_URL=http://orchestrator:$ESPRESSO_ORCHESTRATOR_PORT
+ESPRESSO_SEQUENCER_API_PORT=50000
+ESPRESSO_SEQUENCER1_API_PORT=50001
+ESPRESSO_SEQUENCER2_API_PORT=50002
+ESPRESSO_SEQUENCER3_API_PORT=50003
+ESPRESSO_SEQUENCER4_API_PORT=50004
+ESPRESSO_SEQUENCER_URL=http://espresso-sequencer0:$ESPRESSO_SEQUENCER_API_PORT
+ESPRESSO_SEQUENCER_STORAGE_PATH=/store/sequencer
+ESPRESSO_SEQUENCER_L1_PORT=8545
+ESPRESSO_SEQUENCER_L1_WS_PORT=8546
+ESPRESSO_SEQUENCER_L1_PROVIDER=http://geth:$ESPRESSO_SEQUENCER_L1_PORT
+ESPRESSO_SEQUENCER_L1_WS_PROVIDER=ws://geth:$ESPRESSO_SEQUENCER_L1_WS_PORT
+ESPRESSO_SEQUENCER_L1_USE_LATEST_BLOCK_TAG=true
+ESPRESSO_SEQUENCER_ETH_MNEMONIC="indoor dish desk flag debris potato excuse depart ticket judge file exit"
```

Comment:
Maybe leave a comment that this is the mnemonic that is used for testing in nitro?